### PR TITLE
Update version and recipe

### PR DESCRIPTION
**Pull request by stefan-balke**

### CODE REVIEW

**Comment by stefan-balke** [comment]
@conda-forge-admin, please update circle

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
-  name: bdw-gc
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: gc7_4_2.tar.gz
-  url: https://github.com/ivmai/bdwgc/archive/gc7_4_2.tar.gz
-  sha1: 441777b9865a730c9c804f5f5de2aa9a55fc9419
+  fn: gc-{{ version }}.tar.gz
+  url: https://github.com/ivmai/{{ name }}/releases/download/v{{ version }}/gc-{{ version }}.tar.gz
```
Comment (CJ-Wright) [inline recipe/meta.yaml]: :+1:

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
-  name: bdw-gc
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: gc7_4_2.tar.gz
-  url: https://github.com/ivmai/bdwgc/archive/gc7_4_2.tar.gz
-  sha1: 441777b9865a730c9c804f5f5de2aa9a55fc9419
+  fn: gc-{{ version }}.tar.gz
+  url: https://github.com/ivmai/bdwgc/releases/download/v{{ version }}/gc-{{ version }}.tar.gz
```
Comment (CJ-Wright) [inline recipe/meta.yaml]: `https://github.com/ivmai/bdwgc/archive/v{{ version }}.tar.gz` ?

Comment (stefan-balke) [reply recipe/meta.yaml]: Yes, saw it a second ago since the sha256 didn't work...thanks :)

**Comment by stefan-balke** [comment]
Okay, Circle is building, waiting for the rest to pass and then merge.

**Comment by CJ-Wright** [comment]
**Please** don't build from feedstock branches. These branches get built and shipped!

**Comment by CJ-Wright** [comment]
Should this build on Windows?

**Comment by stefan-balke** [comment]
> Please don't build from feedstock branches. These branches get built and shipped!

What do you mean by that?

> Should this build on Windows?

No.

**Comment by stefan-balke** [comment]
Wait, @CJ-Wright even if I work in a PR the things get pushed?

**Comment by CJ-Wright** [comment]
If the PR is from the feedstock and not your fork YES.
Note that the most recent recipe was added 20 mins ago
https://anaconda.org/conda-forge/bdw-gc/files

**Comment by stefan-balke** [comment]
Oh, oops, sorry, next time I know.

**Comment by CJ-Wright** [comment]
You might want to put in another PR which bumps the build number, just to double check that you have the correct package setup.

**Comment by stefan-balke** [comment]
Okay, just to make sure: adding a +1 here?
https://github.com/conda-forge/bdw-gc-feedstock/blob/master/recipe/meta.yaml#L15

**Comment by CJ-Wright** [comment]
Yep, so it would be `number: 1`

**Pull request merged**